packages/dashboard/src/types.ts

```diff
@@ -30,6 +30,7 @@ export type DashboardWidget<T extends Record<string, unknown> = Record<string, u
   z: number;
   height: number;
   width: number;
+  pageSize?: number;
```

Comment:
We cannot add this here, it affects the public api for all widgets. Instead we should add it to Table Properties - `packages/dashboard/src/customization/widgets/types.ts`

Reply:
Resolved